projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -323,6 +324,12 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
         this.instance.popper.style.width = this.hostWidth;
       });
   }
+
+  private setValueByProps() {
```

Comment:
inline input should win. Why you're not using the same path as other config values that we can define both inline and global? see ngOnChanges